packages/seal/src/client.ts

```diff
@@ -158,20 +158,14 @@ export class SealClient {
 
 	#validateEncryptionServices(services: string[], threshold: number) {
 		// Check that the client's key servers are a subset of the encrypted object's key servers.
-		const serverObjectIdsMap = new Map<string, number>();
-		for (const objectId of this.#serverObjectIds) {
-			serverObjectIdsMap.set(objectId, (serverObjectIdsMap.get(objectId) ?? 0) + 1);
-		}
-		const servicesMap = new Map<string, number>();
-		for (const service of services) {
-			servicesMap.set(service, (servicesMap.get(service) ?? 0) + 1);
-		}
-		for (const [objectId, count] of serverObjectIdsMap) {
-			if (servicesMap.get(objectId) !== count) {
-				throw new InconsistentKeyServersError(
-					`Client's key servers must be a subset of the encrypted object's key servers`,
-				);
-			}
+		if (
+			this.#serverObjectIds.some(
+				(objectId) => count(this.#serverObjectIds, objectId) < count(services, objectId),
```

Comment:
This was an inequality check, but iiuc checking that there are at least as many should be sufficient.